plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/RollbackToSnapshotProcedure.java

```diff
@@ -65,6 +65,6 @@ public void rollbackToSnapshot(ConnectorSession clientSession, String schema, St
     {
         SchemaTableName schemaTableName = new SchemaTableName(schema, table);
         Table icebergTable = catalogFactory.create().loadTable(clientSession, schemaTableName);
-        icebergTable.rollback().toSnapshotId(snapshotId).commit();
+        icebergTable.manageSnapshots().rollbackTo(snapshotId).commit();
```

Comment:
Javadoc suggests `setCurrentSnapshot` should be used. Is there a difference?

Reply:
You are right, `org.apache.iceberg.RollbackToSnapshot#toSnapshotId` does use `setCurrentSnapshot(long)` as well.

Sorry for the confusion.